common/menu.c

```diff
@@ -87,7 +123,7 @@ static void drawEntry(menuEntry_s* me, int off_x, int is_active) {
 
     int border_start_x, border_end_x;
     int border_start_y, border_end_y;
-    color_t border_color = MakeColor(255, 255, 255, 255);
+    color_t border_color = MakeColor(themeCurrent.borderColor.r, themeCurrent.borderColor.g, themeCurrent.borderColor.b, 255);
```

Comment:
MakeColor with a color_t is redundant, same with below.

Reply:
you are correct... changed.